000-cbdb-sandbox/README.md

```diff
@@ -64,7 +84,7 @@ You can now connect to the database and try some basic operations.
 1. Connect to the Docker container from the host machine:
 
     ```shell
-    docker exec -it $(docker ps -q) /bin/bash
+    docker exec -it cbdb-mdw /bin/bash
```

Comment:
With the introduction of the multi-node container configuration, it may take extra time for the cluster to boot. Consider adding a note to inform users that the gpinitsystem process may take additional time to complete.

Reply:
A thought occurred (especially for the multi-node config), to include a `select * from gp_segment_configuration` here to demonstrate the multi-node config in all it's cloudberry glory. Just a thought.

Reply:
For the gp_segment_configuration comment, I made sure to add a note to check the docker logs to see if your database is finished initializing. The gp_segment_configuration output can be found there (you shouldn't need to scroll either). I figured it's best to keep it there than in the README so the README doesn't get too verbose (like should we include output for both single vs multi container deployments to not confuse end users, this was a constant battle I was dealing with when implementing that feature).
https://github.com/antoniopetrole/bootcamp/commit/1bf9949713e086b19cb827542824295ee565b50f

Also added a note to the same commit about how the cluster may take a few minutes to start up (it's really only about 30 seconds on my machine but wanted to be liberal about that timing in case people are using older machines).